opsdataAPI.md

```diff
@@ -1,6 +1,6 @@
 ## OPS-DATA API
 
-The opsdata API offers access to the ParticipantSummary resource and represents an aggregated view of relevant
+The ops-data API offers access to the ParticipantSummary resource and represents an aggregated view of relevant
```

Comment:
Is it ops-data or ops data or Ops Data? :)

Reply:
I think ops-data 🤷‍♂️